boa/src/syntax/lexer/mod.rs

```diff
@@ -3,23 +3,41 @@
 //! The Lexer splits its input source code into a sequence of input elements called tokens, represented by the [Token](../ast/token/struct.Token.html) structure.
 //! It also removes whitespace and comments and attaches them to the next token.
 
+#[macro_use]
 mod comment;
+
 mod cursor;
 pub mod error;
+
+#[macro_use]
 mod string;
 pub mod token;
 
 #[macro_use]
 mod template;
 
+mod number;
+
+mod operator;
+
+mod spread;
+
+mod regex;
+
+mod identifier;
+
 // Temporary disabled while lexer in progress.
```

Comment:
I'm guessing this si no longer true, right?

Reply:
Correct